website/source/docs/agent/http/query.html.markdown

```diff
@@ -44,6 +48,10 @@ its ID if it is created successfully.
 By default, the datacenter of the agent is queried; however, the dc can be
```

Comment:
I never noticed this before, but "dc" should be quoted to `dc` so it becomes monospace.